### ISSUE TITLE
Error: Could not send BinaryInform. Status code 200/408 - maybe fake IMEI isn't working any longer

### ISSUE THREAD

**Issue by redddflash** [ivanmeler/SamFirm_Reborn#16]
The story starts all over again, see output from latest version 0.3.6.7 below. I would assume, the fake IMEI (00000000000000) isn't working any longer. Maybe we would need a new input field for specifying a valid IMEI, or we would need a new database in the background, that could select a valid IMEI on its own?

```
D:\SamFirm>SamFirm_Reborn_0.3.6.7\SamFirm.exe -c -model SM-S918B -region BTB
========== SamFirm Firmware Update Check ==========

Checking firmware for SM-S918B/BTB/S918BXXS3BWKC/S918BOXM3BWJM/S918BXXU3BWJM/S918BXXS3BWKC
Error: Could not send BinaryInform. Status code 200/408
Checking firmware for SM-S918B/BTB/S918BXXU2AWF1/S918BOXM2AWF1/S918BXXU2AWF1/S918BXXU2AWF1
Error: Could not send BinaryInform. Status code 200/408
Could not fetch info for SM-S918B/BTB. Please verify the input or use manual info

```

**Comment by tungtata** [comment]
Yes now you have to use IMEI or SN to get firmware

**Comment by ivanmeler** [comment]
Ill update this tomorrow and add field/parameter to add imei or serial of device you want to download firmware for, hopefully that will stop samsung from changing api for some time

**Comment by ivanmeler** [comment]
Resolved with latest release 0.3.6.8

**Comment by toptop125** [comment]
Again Samfirm is not working please:
SamFirm v0.3.6.0
Checking firmware for SM-R900/CHC/R900ZCU1BWL1/R900CHC1BWL1/R900ZCU1BWL1/R900ZCU1BWL1
Error: Could not send BinaryInform. Status code 200/401
    Authorization failed
Checking firmware for SM-R900/CHC/R900ZCU1AVK7/R900CHC1AVK7/R900ZCU1AVK7/R900ZCU1AVK7
Error: Could not send BinaryInform. Status code 200/401
    Authorization failed
Could not fetch info for SM-R900/CHC. Please verify the input or use manual info
